### ISSUE TITLE
hvcc documentation

### ISSUE THREAD

**Issue by diemildefreude** [enzienaudio/hvcc#15]
I'm looking through the documentation here on github and it seems some things from the old website's documentation are missing, eg. a list of the vanilla PD objects supported by hvcc. Additionally, there are some quirks of hvcc that were not documented on the old site that could be worth documenting (eg. the send/receive objects' right inlet doesn't work, graphical objects like the Toggle are converted into [float] upon compiling).

If I'm mistaken and that documentation is still around here or elsewhere, let me know! Or if there's interest compiling such a document, let's talk about how to go about doing so.

**Comment by giuliomoro** [comment]
yup, reported here https://github.com/enzienaudio/hvcc/issues/1  and there is a pending PR to fix it https://github.com/enzienaudio/hvcc/pull/3

As the repo is unmaintained, I wouldn't expect PRs to ever be merged.

**Comment by diemildefreude** [comment]
Ah, I see the list is there in #3, very good. In addition, it would be good to have a list of some of the aforementioned "quirks". I could help contribute to that, but I'm not sure where to put it- I don't really understand Github.

**Comment by giuliomoro** [comment]
You can start editing the documentation files directly here, then when you try to save it, Github will prompt you and make a copy ("fork") of the  repo on your account. Once you are done with the changes, you can start a "pull request" (PR) to this repo. This is the normal way of doing things, however no one will probably merge your PR here (because this repo is unmaintained). However, someone may come here and find the answers to their question in your PR, which is useful. I have a fork where I merged in some of my fixes, and one Bela-specific commit.
